### ISSUE TITLE
itkLandmarkAtlasSegmentationFilterTest failing

### ISSUE THREAD

**Issue by thewtex** [KitwareMedical/HASI#10]
```
4: Trying inputLandmarks = readSlicerFiducials(inputLandmarksFilename)
4:
4: itk::ExceptionObject (0x1345a50)
4: Location: "readSlicerFiducials"
4: File: /home/matt/src/HASI/test/itkLandmarkAtlasSegmentationFilterTest.cxx
4: Line: 78
4: Description: Unrecognized coordinate system
4:
4:
4:   In /home/matt/src/HASI/test/itkLandmarkAtlasSegmentationFilterTest.cxx,
line 190
4: Exception detected while reading
/home/matt/bin/HASI-Wrap-MinSizeRel/ExternalData/test/Baseline/901-L-A.nrrd :
Could not create IO object for reading file
/home/matt/bin/HASI-Wrap-MinSizeRel/Testing/Temporary/901-L-A.nrrd
4: The file doesn't exist.
4: Filename =
/home/matt/bin/HASI-Wrap-MinSizeRel/Testing/Temporary/901-L-A.nrrd
4: <DartMeasurement name="BaselineImageName"
type="text/string">INVALID_BASELINE_GIVEN</DartMeasurement>
1/1 Test #4: itkLandmarkAtlasSegmentationFilterTest ...***Failed    4.86 sec
```

**Comment by thewtex** [comment]
The test is expecting a line indicating a Slicer coordinate system flip:

https://github.com/KitwareMedical/HASI/blob/9ef8a0b20e3d63e07fd1967e3b556bbc06e1b28c/test/itkLandmarkAtlasSegmentationFilterTest.cxx#L70-L79

But that line does not exist in the current testing inputs, e.g. *901-L.fcsv*:

```
# Markups fiducial file version = 4.10
# CoordinateSystem = 0
# columns = id,x,y,z,ow,ox,oy,oz,vis,sel,lock,label,desc,associatedNodeID
vtkMRMLMarkupsFiducialNode_50,-4.945,-9.225,46.011,0.000,0.000,0.000,1.000,1,1,0,CoFH,Center of Femur Head,vtkMRMLScalarVolumeNode2
vtkMRMLMarkupsFiducialNode_51,-7.424,-9.474,44.711,0.000,0.000,0.000,1.000,1,1,0,FS,Femur Shaft,vtkMRMLScalarVolumeNode2
vtkMRMLMarkupsFiducialNode_52,-5.018,-9.058,47.535,0.000,0.000,0.000,1.000,1,1,0,D,Dent,vtkMRMLScalarVolumeNode2
```

**Comment by thewtex** [comment]
Addressed in #17
